src/Microsoft.OpenApi/Validations/Rules/JsonSchemaRules.cs

```diff
@@ -34,6 +34,23 @@ public static class JsonSchemaRules
 
                     context.Exit();
 
+                    // examples
+                    context.Enter("examples");
+
+                    if (jsonSchema.GetExamples() is { } examples)
+                    {
+                        var examplesCount = examples.Count();
+                        for (int i = 0; i < examplesCount; i++)
```

Comment:
```suggestion
                        for (int i = 0; i < examples.Count; i++)
```

Actually I didn't know the data structure behind the scenes but it has a count property.